_includes/navbar.html

```diff
@@ -54,6 +54,7 @@
                               <a class="dropdown-item" href="cssi.html">CSSI</a>
                               <a class="dropdown-item" href="dbsec.html">DB-Sec</a>
                               <a class="dropdown-item" href="gwic.html">GWiC</a>
+                              <a class="dropdown-item" href="code-n-color.html">Code'n'Color</a>
```

Comment:
This should go under the "Department Led" initiatives on line 65

Reply:
Oops! I'll fix it now.